### ISSUE TITLE
Wallet - Save Address - connect status go endpoint to save address:

### ISSUE THREAD

**Issue by briansztamfater** [status-im/status-react#16994]
# Feature Issue

### User Story

As a user, I want to be able to save an address in the Transaction Status page **so that** I can quickly choose it for future transactions

### Description

This issue is to connect the RPC endpoint for a wallet address to be saved and the data persisted.

Figma design: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1518-358506&mode=design&t=bR18stv43uRLwfK3-4

Visual reference:<img width="180" src="https://github.com/status-im/status-mobile/assets/18485527/bc332a83-87ff-4115-bb57-7405b3e65bda"> 

### Acceptance Criteria

- [ ] status go rpc event should be fired and the data should be persisted
- [ ] toast shows once the event has fired

Feel free to break up this issue in smaller issues if necessary

**Comment by J-Son89** [comment]
I think for now whoever takes this should take this and split it into a few issues. 
For the moment they could start with just the UI and have no backend connection and put this isolated by itself in a flow (i.e you don't need send flow to reach this page yet).

**Comment by ulisesmac** [comment]
@shivekkhurana I was about to take this issue since had no assignees, but I've seen you opened a draft PR

**Comment by shivekkhurana** [comment]
The PR that was merged implements this partly. The implementation is blocked by the network preferences component. 

So opening this issue again.

**Comment by smohamedjavid** [comment]
Closing this issue as we removed the transaction progress screen and adding a new saved address is done through wallet settings.
